Data/IntMap/Internal.hs

```diff
@@ -1740,25 +2433,38 @@ split k t =
 splitLookup :: Key -> IntMap a -> (IntMap a, Maybe a, IntMap a)
```

Comment:
Everything looks like it's already whnf by construction, so it shouldn't change anything. But I can make the change so that it's more explicit